mantidimaging/eyes_tests/reconstruct_window_test.py

```diff
@@ -61,7 +61,7 @@ def test_reconstruction_window_colour_palette_dialog(self):
         self._load_data_set()
 
         self.imaging.show_recon_window()
-        self.imaging.recon.image_view.recon.image = np.empty((512, 512))
+        self.imaging.recon.image_view.imageview_recon.im.image = np.empty((512, 512))
```

Comment:
I think that can be
`self.imaging.recon.image_view.setImage(np.empty((512, 512)))`

Also should probably be `np.zeros` rather than `np.empty`.